server/odc-service/src/main/java/com/oceanbase/odc/config/RestTemplateConfig.java

```diff
@@ -63,8 +63,8 @@ public RestTemplate vpcRestTemplate() {
                         MediaType.APPLICATION_FORM_URLENCODED));
         RestTemplate restTemplate = new RestTemplateBuilder()
                 .additionalMessageConverters(stringHttpMc, fastJsonHttpMc)
-                .setConnectTimeout(Duration.ofSeconds(5))
-                .setReadTimeout(Duration.ofSeconds(5))
+                .setConnectTimeout(Duration.ofSeconds(10))
```

Comment:
5 seconds is suitable for ConnectTimeout

Reply:
fixed